tests/test_config_consistency.py

```diff
@@ -569,12 +569,21 @@ def setup_main(self):
     def test_main(self):
         assert self.req.status_code == 200
 
-        runtime_metrics = [
+        runtime_metrics_gauges = [
```

Comment:
[nit] Should we change the `time.sleep(10)` above to instead be a poller to reduce the expected runtime for these tests?

Reply:
it actually should be handled by the library interface wait time, that is done at the very end of the `setup` phase.

Reply:
I agree, this should be addressed separately but I'm not sure I want to spend time on it immediately since the 10s wait should be sufficient for now without too much downside